src/iocore/cache/unit_tests/test_Stripe.cc

```diff
@@ -0,0 +1,159 @@
+/** @file
+
+  A brief file description
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+#include "main.h"
+
+#include <array>
+#include <ostream>
+
+// Required by main.h
```

Comment:
what is required by math.h?

Reply:
`main.h`, not `math.h`. I copied that comment and the two LoC below it verbatim from another test; I think those symbols are declared in main.h as globals and the test has to define them.